src/messageServer/MessageServerCLI.java

```diff
@@ -0,0 +1,20 @@
+package MessageServer;
+
+import java.util.Scanner;
+
+public class MessageServerCLI {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Willkommen beim Installationsvorgang Ihres MessageServers:");
+		System.out.print("Geben Sie bitte den Port an, �ber welchen Konsumenten und Produzenten den MessageServer erreichen k�nnen ");
+		int portMessageServer = scanner.nextInt();

```

Comment:
Sebastian und ich haben den selben Port (glaube ich, korrigier mich wenn ich falsch liege) in customer und producer jeweils 'serverPort' genannt.
bezüglich verständlichkeit sollte das villeicht einheitlich sein.